### ISSUE TITLE
Can play web video?

### ISSUE THREAD

**Issue by goldssss** [jvanakker/tvOSBrowser#21]
Can't play web video?
hope support play web video
希望支持网络在线视频播放

**Comment by amirvenus** [comment]
This is Github for reporting bugs.

It’d be better to try the code yourself first and then if it doesn’t play videos, you can create an issue here.

**Comment by manwhocode** [comment]
yes the problem really is. it cannot interact with players and as I understand it with iframe objects

**Comment by jalog** [comment]
> Can't play web video?
> hope support play web video
> 希望支持网络在线视频播放

Have you got a browser app running on aTV4K ？

**Comment by manwhocode** [comment]
> > Can't play web video?
> > hope support play web video
> > 希望支持网络在线视频播放
> 
> Have you got a browser app running on aTV4K ？

Open xcode and run it on aTV as simulator
